packages/odf/components/s3-browser/lifecycle-rules-list/LifecycleRulesList.tsx

```diff
@@ -0,0 +1,391 @@
+        title: t('Rule name'),
+        sort: 'ID',
+        transforms: [sortable],
+        id: tableColumnInfo[0],
+      },
+      {
+        title: t('Scope'),
+        id: tableColumnInfo[1],
+      },
+      {
+        title: t('Rule actions'),
+        id: tableColumnInfo[2],
+      },
+      {
+        title: '',
+        id: tableColumnInfo[3],
+      },
+    ],
+    [t]
+  );
+
+  const [columns] = useActiveColumns({
+    columns: tableColumns,
+    showNamespaceOverride: false,
+    columnManagementID: null,
+  });
+
+  return (
+    <VirtualizedTable
+      {...props}
+      aria-label={t('Lifecycle rules list page')}
+      columns={columns}
+      Row={RuleRow}
+    />
+  );
+};
+
+export const LifecycleRulesList: React.FC<LifecycleRulesListProps> = ({
+  obj: { fresh },
+}) => {
+  const { t } = useCustomTranslation();
+
+  const { bucketName } = useParams();
+  const navigate = useNavigate();
+  const launcher = useModal();
+  const { noobaaS3 } = React.useContext(NoobaaS3Context);
+
+  const { data, isLoading, error, mutate } = useSWR(
+    `${bucketName}-${BUCKET_LIFECYCLE_RULE_CACHE_KEY_SUFFIX}`,
+    () => noobaaS3.getBucketLifecycleConfiguration({ Bucket: bucketName }),
+    {
+      shouldRetryOnError: false,
+    }
+  );
+
+  const rules = data?.Rules || [];
+  const loaded = !isLoading && fresh;
+  const noRuleExists = isNoLifecycleRuleError(error) && !rules.length;
+  const createRuleLink = `${BUCKETS_BASE_ROUTE}/${bucketName}/management/lifecycle/create/~new`;
+
+  const [unfilteredRules, filteredRules, onFilterChange] = useListPageFilter(
+    rules,
+    nameFilterOverride
+  );
+
+  React.useEffect(() => {
+    if (!fresh) mutate();
+  }, [fresh, mutate]);
+
+  if (!loaded || (error && !noRuleExists)) {
+    return (
+      <StatusBox
+        loaded={loaded}
+        loadError={!loaded ? '' : error}
+        skeleton={<div className="loading-skeleton--table pf-v5-u-mt-lg" />}
+      />
+    );
+  }
+
+  return (
+    <>
+      <ListPageHeader title={t('Lifecycle rules')}>
+        {!!rules.length && (
+          <ListPageCreateLink to={createRuleLink}>
+            {t('Create lifecycle rule')}
+          </ListPageCreateLink>
+        )}
+      </ListPageHeader>
+      <ListPageBody>
+        {!rules.length ? (
+          <EmptyPage
+            title={t(
+              'No lifecycle rules defined for the objects in your bucket.'
+            )}
+            buttonText={t('Create lifecycle rule')}
+            onClick={() => navigate(createRuleLink)}
+            isLoaded
+            canAccess
+          />
+        ) : (
+          <>
+            <ListPageFilter
+              data={unfilteredRules}
+              loaded
+              hideLabelFilter={true}
+              onFilterChange={onFilterChange}
+              hideColumnManagement={true}
+            />
+            <RulesTable
+              data={filteredRules}
+              unfilteredData={rules}
+              loaded
+              loadError={null}
```

Comment:
```suggestion
              loadError={error}
```
Shouldn't this be error?

Reply:
not needed... for such case we already have `StatusBox` above... `loadError` is a required prop, hence sending `null` instead...